include/ftxui/screen/string.hpp

```diff
@@ -12,12 +13,10 @@ std::wstring to_wstring(T s) {
   return to_wstring(std::to_string(s));
 }
 
-int wchar_width(char32_t);
 int wchar_width(wchar_t);
-int wchar_width_cjk(char32_t);
-int wchar_width_cjk(wchar_t);
 int wstring_width(const std::wstring&);
```

Comment:
These two are also not used in the library - consider moving to deprecated.hpp or removing.

Reply:
Yes, good idea. I will move them to deprecated.